_posts/2017/2017-11-09-devoxx-belguim-day-2.md

```diff
@@ -0,0 +1,88 @@
+make the IT world a better place to work in by making it more welcoming to women.
+IMHO, most of those top 10 should be mandatory to everyone, rno matter if there are woman in your team or not.
+If you're interested in this topic, these 15 minutes are worth viewing.
+And if not, well... YOU SHOULD BE!
+
+## Spring Boot 2.0 Web Applications
+
+The second Spring talk of the day I visited was an introduction to the new features and paradigms of Spring Boot 2.0,
+given by [Stéphane Nicoll](https://twitter.com/snicoll) and [Brian Clozel](https://twitter.com/bclozel).
+Although this session could not keep up with the Josh Long talk with regards to entertainment, the content was also
+quite good.
+If you havn't had a deeper look at the new possibilities that come with Spring Boot 2.0, I would recommend this as a
+good summary.
+
+## How Events Are Reshaping Modern Systems
+
+Akka inventor and Lightbend CTO [Jonas Bonér](https://twitter.com/jboner) gave his view on how event driven architecture
+solves a lot of problems software systems are facing nowadays.
+Ranging from the general concepts of commands end events, touching actor-based systems (guess which one
+{% emoji simple_smile %}), to event sourcing, a lot of topics were covered.
+Not unsurprisingly, the quality of this talk was really great, and it's really worth watching on Youtube!
+
+## Container Orchestration Considerations
+
+In this talk Patrick van der Bleek from Docker tried to give an overview of the considerations you need to make when
+taking Docker containers to production, i.e. how to orchestrate them.
+Beginning with on overview of existing solutions, he afterwards focused on the two main players, Kubernetes and Swarm.
+This is where the quality of the talk began to drop, as he was, imho, (natually) so strongly biased towards Swarm that
+a lot of his comparisons were plainly misleading.
+Altogether, if you're looking for an orchestration tool, you will still get some valuable input from this talk.
+Just keep his bias in mind and also watch a talk from a Kubernetes advocate. {% emoji simple_smile %}
+
+## Common API security pitfalls
+
+As the final talk of the second day, I selected [Philippe De Ryck](https://twitter.com/PhilippeDeRyck) presenting his
+personal top 11 web application / API security fallacies.
+Every developer providing web APIs should at least have a look at the slides, to make sure you know about the possible
+weaknesses your API exposes.
+Nice talk, not too challenging, and therefore a good fit for the end of the agenda for Thursday.
+
+## Summary
+
+To round up this day, which again had an amazing density of valuable information to me, the devoxxians were invited to
+see [Blade Runner 2049](http://www.imdb.com/title/tt1856101/), which quite a lot did.
+It made it a nice evening of a very long and exhausting day, which once again was really worth visiting the devoxx.
```

Comment:
Double "it" at the beginning of the sentence. Maybe "The movie made it ... " or "This made it..."